pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -2604,7 +2579,8 @@ in
   };
 
   tmux-complete-vim = super.tmux-complete-vim.overrideAttrs {
-    dependencies = with self; [ nvim-compe ];
+    # Vim plugin with optional nvim-compe lua module
+    doCheck = false;
```

Comment:
Technically does nothing until https://github.com/NixOS/nixpkgs/pull/352277 is implemented.